docs/terraform.md

```diff
@@ -32,6 +32,7 @@
 | name | Solution name, e.g. 'app' or 'jenkins' | `string` | `null` | no |
 | namespace | Namespace, which could be your organization name or abbreviation, e.g. 'eg' or 'cp' | `string` | `null` | no |
 | regex\_replace\_chars | Regex to replace chars with empty string in `namespace`, `environment`, `stage` and `name`.<br>If not set, `"/[^a-zA-Z0-9-]/"` is used to remove all characters other than hyphens, letters and digits. | `string` | `null` | no |
+| scope\_role\_to\_namespace\_enabled | Enables scoping the created role to the namespace instead of an individual service account. | `bool` | `false` | no |
```

Comment:
IMO kube naming style will be much better here, something like this `is_namespaced_scope`, but need to discuss with others it's just suggestion

Reply:
The pattern we've been following in our CP modules is `*_enabled` -- I believe we'd want to keep with that. Let's discuss in #contributors